com.civicrm.hrjobroles/js/test/filters/option-group-filter_test.js

```diff
@@ -0,0 +1,59 @@
+define([
+    'common/angular',
+    'common/angularMocks',
+    'job-roles/filters/option-group-filter',
+    'job-roles/filters/filters',
```

Comment:
There is no need to declare this requirejs module here, it's already being declared inside `job-roles/filters/option-group-filter`